extensions/amp-analytics/0.1/vendors.js

```diff
@@ -644,6 +644,76 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'linkpulse': {
+    'vars': {
+      'id': '',
+      'pageUrl': 'CANONICAL_URL',
+      'title': 'TITLE',
+      'section': '',
+      'channel': 'amp',
+      'type': '',
+      'host': 'pp.lp4.io',
+      'empty': '',
+    },
+    'requests': {
+      'base': 'https://${host}',
+      'pageview': '${base}/p?i=${id}' +
+                '&r=${documentReferrer}' +
+                '&p=${pageUrl}' +
+                '&s=${section}' +
+                '&t=${type}' +
+                '&c=${channel}' +
+                '&mt=${title}' +
+                '&_t=amp' +
+                '&_r=${random}',
+      'pageload': '${base}/pl?i=${id}' +
```

Comment:
what's the difference between pageview and pageload? Can you not combine the two?